src/main/java/uk/gov/hmcts/reform/sandl.snlevents/config/JpaTestConfiguration.java

```diff
@@ -0,0 +1,100 @@
+package uk.gov.hmcts.reform.sandl.snlevents.config;
```

Comment:
You can move this file to test/java/... directory structure as it's related to tests.
Named Qualifiers seems to be not needed as it's taking Beans by type, but better check if you want to go that way.
You can change method names for those beans, to show what you are returning, as you do in fact create stubs - but that just me being picky ;)

Reply:
I have moved the file and changed the names, but regarding named qualifiers. There was some issue with finding EntityManagerFactory and providing it with named qualifier helped. We can take a look if you have an idea what could be wrong.